### PR TITLE
MISUV-4607 : Fixed Table in Tax Year Summary Page

### PR DESCRIPTION
**Pull request by AshokGitAccount**

### CODE REVIEW

File: app/views/TaxYearSummary.scala.html [hunk 1]
```diff
@@ -368,6 +368,7 @@ <h1 class="govuk-heading-xl" id="heading">
                 )
             )))
         }
+        @if(!errorPresent(model)) {
```
Comment (avelx) [inline app/views/TaxYearSummary.scala.html:371]: @AshokGitAccount  - I would assume this was not part of the IT test originally (was not covered ) before we started working on this story?

Comment (avelx) [reply app/views/TaxYearSummary.scala.html:371]: can you add IT test to cover this scenario ?

Comment (AshokGitAccount) [reply app/views/TaxYearSummary.scala.html:371]: @avelx - added check for forecast table to not be displayed if there are error messages in the existing IT

Comment (avelx) [reply app/views/TaxYearSummary.scala.html:371]: @AshokGitAccount - do we need to update spec for agent as well?

**Pull request merged**